src/main.rs

```diff
@@ -158,7 +160,58 @@ pub fn ask_to_run(mut cmd: Command, ask: bool, text: &str) {
     }
 }
 
-fn build_sysroot(auto: bool, target: &str, rustc_version: &VersionMeta) -> PathBuf {
+/// Returns the sanitizer to use.
+/// Ok(true) => Use sanitizer "string"
+/// Ok(false) => Use no sanitizer
+///
+/// # Errors
+/// Err(err) => There was an error when trying to get the list of supported sanitizers
+pub fn sanitizer(target: &str, san: &str) -> Result<bool> {
```

Comment:
```suggestion
/// Returns whether the given sanitizer is supported.
///
/// # Errors
/// Err(err) => There was an error when trying to get the list of supported sanitizers
pub fn is_sanitizer_supported(target: &str, san: &str) -> Result<bool> {
```
Is this correct? The old doc comment confused me.